### PR TITLE
Fix text overflow in Source Interface replies 

### PR DESCRIPTION
**Pull request by ghost**
## Status

Ready for review

## Description of Changes

I couldn't find an issue for this. I hope it's alright if I fixed it. The issue is when long text is displayed in the "Read Replies" box, the text would overflow out of the box. This patch fixes it.

## Testing

1. make dev
2. Go to http://localhost:8080.
3. Click on "Log in".
4. Enter the test codename for last source (3/3) in the test file.
5. The long word no longer overflow out of the "Read replies" box with this patch.

## Deployment

N/A

### If you made changes to the server application code:

- [ ] Linting (`make lint`) and tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you added or removed a file deployed with the application:

- [ ] I have updated AppArmor rules to include the change

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

Choose one of the following:

- [ ] I have opened a PR in the [docs repo](https://github.com/freedomofpress/securedrop-docs) for these changes, or will do so later
- [ ] I would appreciate help with the documentation
- [X] These changes do not require documentation

### CODE REVIEW

File: securedrop/sass/source.sass [hunk 1]
```diff
@@ -143,6 +143,7 @@ p#codename
     margin-bottom: 1em
     border-radius: 5px
     position: relative
+    word-wrap: break-word
```
Comment (legoktm) [inline securedrop/sass/source.sass]: According to <https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap>, the official name is "overflow-wrap" and "word-wrap" is an alias for legacy reasons, could you switch to overflow-wrap?

**Review by legoktm — CHANGES_REQUESTED** [review]
Hi @zishanmirza, thanks for the PR! I tested it using your test plan and functionally it looks good to me.

Before:
Screenshot 2022-02-22 at 16-11-43 SecureDrop Protecting Journalists and Sources

After:
Screenshot 2022-02-22 at 16-12-32 SecureDrop Protecting Journalists and Sources

Besides the one inline comment, I have one small nitpick: could you edit the first line of your commit message to be active and specify what component is being fixed. Like, "Fix text overflow in Source Interface replies".

File: securedrop/sass/source.sass (hunk 1 above)
Comment (ghost) [reply securedrop/sass/source.sass]: Sure, I can switch "word-wrap" to "overflow-wrap".

**Comment by ghost** [comment]
I have changed the commit message title.

**Review by legoktm — APPROVED** [review]
LGTM, thanks for the contribution!

**Pull request merged**